### PR TITLE
Refactor Jiva controller and replica clients

### PR DESCRIPTION
**Pull request by prateekpandey14**
**What this PR does / why we need it**:
This PR will refactor the jiva controller and replica client codes 
and move inside to `pkg/client/jiva`.
This will help us to write CTL commands related to Volume and snapshots.

### CODE REVIEW

File: pkg/client/jiva/model.go [hunk 1]
```diff
@@ -50,7 +49,7 @@ type InfoReplica struct {
 	Chain           []string            `json:"chain"`
 	Disks           map[string]DiskInfo `json:"disks"`
 	RemainSnapshots int                 `json:"remainsnapshots"`
-	RevisionCounter string              `json:"revisioncounter"`
+	RevisionCounter int64               `json:"revisioncounter"`
```
Comment (kmova) [inline pkg/client/jiva/model.go]: why is this data type change required?

File: pkg/client/jiva/controller_client.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	}
+
+	if len(volume.Data) == 0 {
+		return nil, errors.New("No volume found")
+	}
+
+	return &volume.Data[0], nil
+}
+func (c *ControllerClient) Get(path string, obj interface{}) error {
+	//	resp, err := http.Get(c.address + path)
```
Comment (utkarshmani1997) [inline pkg/client/jiva/controller_client.go]: remove commented code

File: pkg/client/jiva/model.go (hunk 1 above)
Comment (prateekpandey14) [reply pkg/client/jiva/model.go]: It should be `string`, i think while refactoring/replace(duplicate struct), had deleted the struct which was getting used before.

Comment (prateekpandey14) [reply pkg/client/jiva/model.go]: done !

File: pkg/client/jiva/controller_client.go (hunk 2 above)
Comment (prateekpandey14) [reply pkg/client/jiva/controller_client.go]: done !!

**Pull request merged**